server/app/views/admin/programs/ProgramIndexView.java

```diff
@@ -263,7 +263,7 @@ private ImmutableList<Modal> buildPublishSingleProgramModals(
                           "flex", "flex-col", StyleUtils.responsiveMedium("flex-row"), "py-4");
 
               return Modal.builder()
-                  .setModalId(program.adminName() + "-publish-modal")
+                  .setModalId("publish-modal-" + program.slug())
```

Comment:
Blocking: Recommend creating a method you can pass a program to that will build the modal ID so it doesn't get out of sync between the spots you need it.

Reply:
good idea. done.